### DISCUSSION TITLE
【来点情绪】CODM策划再不削BP50和MG42我让你飞起来！

### DISCUSSION THREAD

**Discussion by GanlinSama** [share121/inter-knot#1232] [其他]
如题，楼主最近在排位中被手持BP50的 _大肘子_ 打破防了
搞什么啊，我短剑**贴脸提前枪**被BP50反杀了(ノ｀Д)ノ

**Comment by gaojiaobei** [comment]
一个月前MG42很强，现在回来，MG42还是很强

**Comment by YukariZZZ** [comment]
刚抽出BP50神话，策划你敢削弱我就让你飞起来！距离远了都秒不了人，还有那个穿墙十几发了都打不死。

**Comment by GanlinSama** [comment]
楼上什么人才？CP点留着抽JAK12神话啊😜
